storage/innobase/xtrabackup/src/backup_copy.cc

```diff
@@ -1746,85 +1851,19 @@ copy_back()
 
 	it = datadir_iter_new(".", false);
```

Comment:
few lines above we copy ibdata1, undo and redo logs. We should mention in blueprint that we don't copy them in parallel.